Globally standardize "the Microsoft Edge Add-ons website" (#3900)
MicrosoftDocs/edge-developer: microsoft-edge/extensions/developer-guide/native-messaging.md

* Package your extension and the host together.  When a user installs the package, both the extension and the host are installed.

> * Or, install your extension from the [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com) website, and your extension prompts users to install the host.<!-- todo: consistent voice: who is referred to, Dev or user? -->

Reviewer: ```suggestion
* Or, publish your extension to the [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com) website.  When  a user installs the extension, the extension prompts the user to install the host.
```
